File: app/src/main/java/com/yuyu/module/chain/ChainedTextView.java
package com.yuyu.module.chain;

import android.content.Context;
import android.widget.TextView;

public class ChainedTextView extends TextView {

    public ChainedTextView(Context context) {
        super(context);
    }

    public ChainedTextView setHeightTo(int pixels) {
        setHeight(pixels);
        return this;
    }

}

File: app/src/main/java/com/yuyu/module/chain/ChainedTextView.kt
package com.yuyu.module.chain

import android.content.Context
import android.widget.TextView

class ChainedTextView(context: Context) : TextView(context) {

    fun setHeightTo(pixels: Int): ChainedTextView {
        height = pixels
        return this
    }

}

/*
package com.yuyu.module.chain;

import android.content.Context;
import android.widget.TextView;

public class ChainedTextView extends TextView {

    public ChainedTextView(Context context) {
        super(context);
    }

    public ChainedTextView setHeightTo(int pixels) {
        setHeight(pixels);
        return this;
    }

}
*/
